contracts/interfaces/IAxelarGMPTracker.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+interface IAxelarGMPTracker {
+    event GMPTracker(
```

Comment:
perhaps rename it to `GMPVolume`? `emit GMPVolume(...)` sounds a bit more natural to me.